### PR TITLE
[debian] fix build depends

### PR DESCRIPTION
**Pull request by wsnipex**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Source: xbmc-audioencoder-vorbis
 Priority: extra
 Maintainer: Arne Morten Kvarving <spiff@xbmc.org>
-Build-Depends: debhelper (>= 8.0.0), cmake, xbmc-addon-dev, automake, autoconf, libtool, libvorbis-dev
+Build-Depends: debhelper (>= 8.0.0), cmake, xbmc-addon-dev, xbmc-audio-dev, make, libtool, libvorbis-dev
```
Comment (jmarshallnz) [inline debian/control:4]: what's in the xbmc-audio-dev package that isn't in the xbmc-addon-dev package?

(i.e. what are these - just the bindings?)

Comment (wsnipex) [inline debian/control:4]: yes, just the bindings. in this case the audioenc\* headers

Comment (jmarshallnz) [inline debian/control:4]: Do we need more than one package if all there is are headers?

Comment (wsnipex) [inline debian/control:4]: Need no, but its good practice. Currently we have xbmc-pvr-dev, xbmc-audio-dev, xbmc-screensaver-dev, xbmc-visualization-dev

Comment (jmarshallnz) [inline debian/control:4]: Fair enough - merge 'em whenever you like.

Comment (wsnipex) [inline debian/control:4]: I don't have permission ;)

Comment (jmarshallnz) [inline debian/control:4]: Ah - will fix :)

Comment (jmarshallnz) [inline debian/control:4]: Try now :)

**Pull request merged**